### PR TITLE
add fabric8-env

### PR DESCRIPTION
**Pull request by nurali-techie**
New environment service in OSIO.

### CODE REVIEW

File: dsaas-services/f8-env.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+services:
+- hash: fead97763ad02a318ccce057cf84c0c1c1681480
+  name: fabric8-env
+  path: /openshift/f8env.app.yaml
+  url: https://github.com/fabric8-services/fabric8-env/
+  hash_length: 6
```
Comment (nurali-techie) [inline dsaas-services/f8-env.yaml]: I see, some has hash_length 6 and some has 7.  What it should be and where it's used ??

**Comment by ppitonak** [comment]
[e2e-test-released]

**Comment by ppitonak** [comment]
[e2e-test-released] [e2e-test-beta]

**Comment by ppitonak** [comment]
@aslakknutsen @alexeykazakov  feel free to force-push. We are investigating the root cause of test failures but it's definitely not related to this PR.

**Pull request merged**